spinel/codec.py

```diff
@@ -183,6 +186,7 @@ def parse_field(cls, payload, spinel_format):
             'L': cls.parse_L,
             'l': cls.parse_l,
             '6': cls.parse_6,
+            'Q': cls.parse_Q,
```

Comment:
This function and the one below seem to unpack using the spinel primitive type format `spinel_format`. We use `'X'` for `uint64_t` type, not `'Q'`.

Reply:
Thanks! Updated.